backend/app/Http/Controllers/User/ShowEditUserController.php

```diff
@@ -93,6 +109,28 @@ public function __invoke(): View|Factory
             'name' => $id->label(), //学部名
         ], $uuFacultyEnum);
 
+        /**
+         * ユーザー情報追加取得
+         */
+        //学科
+        /** @var UUMajor */
+        $uuMajor = UUMajor::where('id', $userInfo->u_u_major_id)->first();
+        $userInfo->u_u_faculty_id = $uuMajor->uu_faculty_id;
+
+        $userAreas = Area::whereIn('id', [$userInfo->birth_area_id, $userInfo->live_area_id])->get();
+        // 在住と出身が同じだった場合、返り値1つなので
+        /** @var Area */
+        $userBirthArea = $userAreas->first(fn (Area $area) => $area->id === $userInfo->birth_area_id);
+        /** @var Area */
+        $userLiveArea = $userAreas->first(fn (Area $area) => $area->id === $userInfo->live_area_id);
+        // userにbirthとliveプロパティを追加
+        \Log::debug($userAreas);
+        $user->live_prefecture_id = $userLiveArea->prefecture_code;
+        $user->live_country_id =  $userLiveArea->country_code;
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Access to an undefined property App\Models\User::$live_country_id.